app/controllers/builds_controller.rb

```diff
@@ -9,15 +9,15 @@ class BuildsController < ApplicationController
   before_action :find_build, only: [:show, :build_docker_image, :edit, :update]
 
   def index
-    search = params[:search]&.to_unsafe_h&.except(:time_format)&.select { |_, v| v.present? } || {}
     builds = scope
-    if status = search.delete(:status)
+
+    if status = params[:search]&.delete(:status)&.presence
```

Comment:
maybe simpler to do `search = params[:search] || ActionPack::Parameters.new` and then not deal with all the nils